types/dragscroll/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
+        "baseUrl": "../",
+        "typeRoots": [
+            "../"
+        ],
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true,
+        "strictFunctionTypes": false
```

Comment:
Meant to be true?

Reply:
To clarify, in your PR template you have "`strictFunctionTypes` set to `true`" checked, but it looks like it's `false` here.

Reply:
Oh wow, yea that completely went over my head. I've made the needed change